File: 231-power-of-two/power-of-two.cpp
class Solution {
public:
    bool isPowerOfTwo(int n) {

       if(n<=0||(n%2!=0&&n!=1))
       return 0;

     float t=log2(n);
     int t1=log2(n);
     cout<<t<<" "<<t1;

     return (t==t1);

    }
};

File: 231-power-of-two/power-of-two.py
class Solution(object):
    def isPowerOfTwo(self, n):
        """
        :type n: int
        :rtype: bool
        """
        if(n<=0):
            return False
        while(n!=1):
            if(n%2==1):
                return False
            n=n/2
        
        return True;